### PR TITLE
fix(font): Add MS-Yahei font to playlist and subtitle module on Windows

### PR DESCRIPTION
**Pull request by huzhenghui37**

### CODE REVIEW

File: src/renderer/components/PlayingView/RecentPlaylist.vue [hunk 1]
```diff
@@ -273,7 +273,7 @@ export default {
     width: 90%;
     .top {
       margin-top: 1px;
-      font-family: Avenir-Heavy;
+      font-family: Avenir-Heavy,Arial,"Microsoft YaHei";
```
Comment (tomasen) [inline src/renderer/components/PlayingView/RecentPlaylist.vue]: 逗号之后应该有空格

File: src/renderer/components/PlayingView/RecentPlaylist.vue [hunk 2]
```diff
@@ -283,7 +283,7 @@ export default {
       text-overflow: ellipsis;
       white-space: nowrap;
 
-      font-family: Avenir-Heavy;
+      font-family: Avenir-Heavy,Arial,"Microsoft YaHei";
```
Comment (tomasen) [inline src/renderer/components/PlayingView/RecentPlaylist.vue]: 如上

File: src/renderer/components/PlayingView/RecentPlaylistItem.vue [hunk 3]
```diff
@@ -292,7 +292,7 @@ $border-radius: 3px;
     white-space: nowrap;
 
     z-index: 100;
-    font-family: Avenir-Heavy;
+    font-family: Avenir-Heavy,Arial,"Microsoft YaHei";
```
Comment (tomasen) [inline src/renderer/components/PlayingView/RecentPlaylistItem.vue]: 如上

**Review by tomasen — CHANGES_REQUESTED** [review]

File: src/renderer/components/PlayingView/RecentPlaylist.vue (hunk 1 above)
Comment (huzhenghui37) [reply src/renderer/components/PlayingView/RecentPlaylist.vue]: Added

File: src/renderer/components/PlayingView/RecentPlaylist.vue (hunk 2 above)
Comment (huzhenghui37) [reply src/renderer/components/PlayingView/RecentPlaylist.vue]: Added

File: src/renderer/components/PlayingView/RecentPlaylistItem.vue (hunk 3 above)
Comment (huzhenghui37) [reply src/renderer/components/PlayingView/RecentPlaylistItem.vue]: Added

**Review by tomasen — APPROVED** [review]

**Pull request merged**